Cargo.lock

```diff
@@ -5341,18 +5393,18 @@ dependencies = [
 
 [[package]]
 name = "strum"
-version = "0.26.1"
+version = "0.25.0"
```

Comment:
something is weird here ... looks like your attempt to upgrade strum isn't put into the lock file properly? Maybe something else resolves wrong or you didn't run the `cargo make desktop` after?